Lib9c/BlockPolicySource.cs

```diff
@@ -85,8 +83,7 @@ bool ignoreHardcodedPolicies
                 maxTransactionsPerBlock: maximumTransactions,
                 maxBlockBytes: MaxBlockBytes,
                 maxGenesisBytes: MaxGenesisBytes,
-                ignoreHardcodedPolicies: ignoreHardcodedPolicies,
-                permissionedMiningPolicy: permissionedMiningPolicy,
```

Comment:
Is this intended? It seems required yet.

https://github.com/planetarium/lib9c/blob/0dd64bdf63c241edf46551521067503f730c111e/Lib9c/BlockPolicy.cs#L71